internal/services/session_configuration.go

```diff
@@ -0,0 +1,19 @@
+package services
+
+import (
+	"time"
+
+	"github.com/kelseyhightower/envconfig"
+)
+
+// SessionConfiguration defines a struct with required environment variables for the session
+type SessionConfiguration struct {
+	MaxAge     int           `envconfig:"SESSION_MAX_AGE" default:"604800"`
+	JWTExpires time.Duration `envconfig:"SESSION_JWT_EXPIRES_AT" default:"24h"`
+	Issuer     string        `envconfig:"SESSION_ISSUER" default:"http://authorization-service:5555/"`
+}
```

Comment:
this logic belongs to the service that handles the creation of the session, which should be the OIDC one